transformer_engine/pytorch/ops/_common.py

```diff
@@ -10,13 +10,13 @@
 import torch
 
 from transformer_engine_torch import FP8TensorMeta
-from ..fp8 import FP8GlobalStateManager
-from ..tensor.float8_tensor import Float8Tensor
-from ..utils import (
+from transformer_engine.pytorch import torch_version
+from transformer_engine.pytorch.fp8 import FP8GlobalStateManager
+from transformer_engine.pytorch.tensor.float8_tensor import Float8Tensor
+from transformer_engine.pytorch.utils import (
```

Comment:
We should have a consistent style and we use relative imports everywhere else: https://github.com/NVIDIA/TransformerEngine/blob/2834e4ad83b86fff086b2882737e2c55f99d994f/transformer_engine/pytorch/module/linear.py#L22-L59

```suggestion
from .. import torch_version
from ..fp8 import FP8GlobalStateManager
from ..tensor.float8_tensor import Float8Tensor
from ..utils import (
```

That said, absolute paths are considered better style ([PEP 8](https://peps.python.org/pep-0008/#imports), [Google style guide](https://google.github.io/styleguide/pyguide.html#224-decision)). If we change style, we should change it everywhere in a single PR with no functional changes.